.github/workflows/centaur_integration_tests.yml

```diff
@@ -0,0 +1,61 @@
+name: 'Centaur Integration Tests'
+
+#This github action runs all of Cromwell's integration tests.
+
+#This is what shows up in the github workflows page as the title. 
+run-name: ${{ github.actor }} Centaur Testing.
+
+#What will trigger the workflow to run. 
+on:
+  workflow_dispatch: #Manual trigger from GitHub UI
+  push: #git push
+
+permissions: 
+  contents: read
+
+jobs:
+  centaur-tests:
```

Comment:
As above, since these aren't all Centaur tests, can we use a different label here? Just `run-test` or something?